Index: Day_35_Keys_Authentication_and_Environment/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import requests\n\napi_key = \"f86e6a4afd3eb7c5fc434f4e601e9efe\"\n\nendpoint = \"https://api.openweathermap.org/data/2.5/onecall\"\n\nparameters = {\n    \"lat\": 55.53,\n    \"lon\": 28.66,\n    \"appid\": api_key,\n}\n\nresponse = requests.get(url=endpoint, params=parameters)\nresponse.raise_for_status()\nprint(response.json())\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Day_35_Keys_Authentication_and_Environment/main.py b/Day_35_Keys_Authentication_and_Environment/main.py
--- a/Day_35_Keys_Authentication_and_Environment/main.py	(revision 7c64180054b5a9d5cff9203cadd5aa8f4e63a7b8)
+++ b/Day_35_Keys_Authentication_and_Environment/main.py	(date 1678827386731)
@@ -13,3 +13,4 @@
 response = requests.get(url=endpoint, params=parameters)
 response.raise_for_status()
 print(response.json())
+
diff --git a/.idea/shelf/Uncommitted_changes_before_Checkout_at_08_03_2023_11_53_[Changes]/shelved.patch b/.idea/shelf/Uncommitted_changes_before_Checkout_at_08_03_2023_11_53_[Changes]/shelved.patch
deleted file mode 100644
