Index: Day_29_Buiding_a_password_manager_GUI_App_with_Tkinter/main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Day_29_Buiding_a_password_manager_GUI_App_with_Tkinter/main.py b/Day_29_Buiding_a_password_manager_GUI_App_with_Tkinter/main.py
new file mode 100644
--- /dev/null	(date 1677908319064)
+++ b/Day_29_Buiding_a_password_manager_GUI_App_with_Tkinter/main.py	(date 1677908319064)
@@ -0,0 +1,18 @@
+from tkinter import *
+
+# ---------------------------- PASSWORD GENERATOR ------------------------------- #
+
+# ---------------------------- SAVE PASSWORD ------------------------------- #
+
+# ---------------------------- UI SETUP ------------------------------- #
+window = Tk()
+
+window.title("Password Manager")
+window.config(padx=20, pady=20)
+
+canvas = Canvas(width=200, height=200)
+password_image = PhotoImage(file="logo.png")
+canvas.create_image(100, 100, image=password_image)
+canvas.pack()
+
+window.mainloop()
